### DISCUSSION TITLE
gRPC流量和XTLS有没有特征上的区别?

### DISCUSSION THREAD

**Discussion by Dawn-ATCG** [XTLS/Xray-core#653] [Q&A]
我是用 VLESS+tls+gRPC 延迟显著低于 VLESS+tcp+XTLS. 请问一下这两个流量上有没有特征区别, XTLS我稳定用了半年多了没有被墙, gRPC会不会什么特征导致更容易被墙? (我已经配置了nginx前置进行伪装站点)

**Comment by joyoner** [comment]
同问

**Reply by GreatMichaelLee** [reply]
都是over tls，我觉得没啥区别，要说特征，如果你看4k视频之类的，某个非门户网站IP的超大流量访问，就是最大的特征
